src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -83,14 +88,29 @@ def validate(self, data):
 
 
 @region_silo_endpoint
+@extend_schema(tags=["Releases"])
 class ReleaseThresholdStatusIndexEndpoint(OrganizationReleasesBaseEndpoint, EnvironmentMixin):
     owner: ApiOwner = ApiOwner.ENTERPRISE
     publish_status = {
-        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "GET": ApiPublishStatus.PUBLIC,
     }
 
+    @extend_schema(
+        operation_id="Retrieve status of a release threshold (Alpha)",
+        parameters=[GlobalParams.ORG_SLUG],
+        request=None,
+        responses={
+            200: inline_sentry_response_serializer(
+                "ReleaseThresholdStatusResponse", dict[str, list[EnrichedThreshold]]
```

Comment:
Did you mean to say that the fields in `SerializedThreshold` may not be in the response? The endpoint example is failing validation b/c it's missing some fields in `SerializedThreshold`

https://github.com/getsentry/sentry/blob/c8a00d1cf64a349276060458ca51dab9e2538f3e/src/sentry/api/endpoints/release_thresholds/types.py#L5-L24

If so, it needs to be initialized like `class SerializedThreshold(TypedDict, total=False):`.